libraries/AP_Motors/AP_MotorsHeli_RSC.cpp

```diff
@@ -231,18 +241,39 @@ void AP_MotorsHeli_RSC::output(RotorControlState state)
 
             // control output forced to zero
             _control_output = 0.0f;
+            
+            // governor is forced to disengage status and ensure governor outputs are reset
+            _autothrottle = false;
+            _governor_engage = false;
+            _governor_fault = false;
+            _governor_output = 0.0f;
+            _governor_torque_reference = 0.0f;
             break;
 
         case ROTOR_CONTROL_IDLE:
             // set rotor ramp to decrease speed to zero
             update_rotor_ramp(0.0f, dt);
 
-            if (_in_autorotaion) {
-                // if in autorotation and using an external governor, set the output to tell the governor to use bailout ramp
-                _control_output = constrain_float( _rsc_arot_bailout_pct/100.0f , 0.0f, 0.4f);
+            // set rotor control speed to engine idle and ensure governor is not engaged, if used
+            _governor_engage = false;
+            _governor_fault = false;
+            _governor_output = 0.0f;
+            _governor_torque_reference = 0.0f;
+            if (!_autothrottle) {
+                if (_in_autorotaion) {
```

Comment:
@bnsgeyer "autorotation" is spelled incorrectly here. This came from the autorotation code and was merged here during the rebase on master. Should make sure this if() statement executes correctly.

Reply:
The autorotation code only applies for SITL.   Additionally it only applies to the SETPOINT RSC mode because this particular if statement is placing the idle throttle output in the autorotation PWM that ESC's want to see so they spool up immediately rather than slow start up.  I'm pretty confident that I merged it correctly.  I do want you to test this branch to ensure the autothrottle functionality has not been altered by the merge with master.

Reply:
Great will do that. I saw misspelling and thought maybe was accidental and would not execute properly. Have not looked at rest of auto code to see if misspelled thru out.  Hopefully can build and test tomorrow on master.